proto/src/main/proto/bakdata/kafka/proto/v1/deadletter.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package bakdata.kafka.proto.v1;
```

Comment:
I don't know what is best practice here but why not include com?

Reply:
Following the example here: https://docs.buf.build/best-practices/style-guide - not explicitly stated, however.